Bootstraps/Symfony.php

```diff
@@ -49,16 +49,24 @@ public function getApplication()
         $appAutoLoader = './app/autoload.php';
         if (file_exists($appAutoLoader)) {
             require $appAutoLoader;
-        } else {
+        } elseif (file_exists('./vendor/autoload.php')) {
             require './vendor/autoload.php';
         }
+        if(class_exists('\AppKernel')) {
+            $app = new \AppKernel($this->appenv, $this->debug);
+        } else {
+            if(file_exists('./app/bootstrap.php.cache') && file_exists('./app/AppKernel.php')) {
```

Comment:
Ah, maybe you just upgraded from a older versions or used a version pre November 2015, where AppKernel is not in the composer.json (like in the README.md of this repo noted). So, then let us require that file if it exists. But please check both files individually, app/AppKernel and app/bootstrap.php.cache. WDYT?